### DISCUSSION TITLE
[SOLVED] Use published field

### DISCUSSION THREAD

**Discussion by ghost** [martinrotter/rssguard#1012] [Q&A] [answered]
How can I switch RSSGuard from using `updated` to using `published` for Atom feeds?

The filter provided here does not work ~~even after fixing the syntax errors~~:

https://github.com/martinrotter/rssguard/blob/2e2aba943cb555213a1924e1e2e3d9dca6994bb7/resources/docs/Documentation.md?plain=1#L243C1-L243C74

**Comment by martinrotter** [comment]
Hi, the filter seems to be about right, please can you give some URL of ATOM feed you use? I will try it.

**Reply by ghost** [reply]
Correction, it _does_ work  when fetching new articles — my bad. I had to refetch the feed.

I wanted to run it on already downloaded articles from f.ex. <https://www.youtube.com/feeds/videos.xml?channel_id=UCy1Ms_5qBTawC-k7PVjHXKQ>

**Reply by martinrotter** [reply, marked as answer]
No, it is not possible to run this filter and make it work on already fetched articles, the functionality of article filters in general allows that but this specific filter will not work because the "published" information is not available for already fetched articles.
